windows/tools/devicemon/Main.cxx

```diff
@@ -63,10 +68,12 @@ main(int argc, char* argv[])
         }
     }
 
-    windows::devices::DevicePresenceMonitor monitor{deviceGuid.value(), [](auto&& presenceEvent, auto&& deviceName) {
-        const auto presenceEventName = magic_enum::enum_name(presenceEvent);
-        std::cout << deviceName << " " << presenceEventName << std::endl;
-    }};
+    windows::devices::DevicePresenceMonitor monitor{ deviceGuid.value(),
+                                                     [](auto&& presenceEvent, auto&& deviceName) {
+                                                        const auto presenceEventName = magic_enum::enum_name(presenceEvent);
+                                                        std::cout << deviceName << " " << presenceEventName << std::endl;
+                                                     } 
+                                                    };

```

Comment:
Is this change intentional?

Reply:
The auto formatting was causing the tabbing to look weird so I changed it to this.

Reply:
I changed it back to how it was formatted before